src/wallet/wallet.h

```diff
@@ -584,6 +584,7 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
                   int sighash_type = 1 /* SIGHASH_ALL */,
                   bool sign = true,
                   bool bip32derivs = true,
+                  bool finalize = true,
```

Comment:
nit: could also add doxygen documentation (a few lines above) for the newly introduced parameter

Reply:
Done